compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -4944,7 +4945,8 @@ declare_lint! {
     ///
     /// struct MyType;
     /// MyType.hello();
-    /// ```
+    #[cfg_attr(bootstrap, doc = "```")]
+    #[cfg_attr(not(bootstrap), doc = "```")]
```

Comment:
You shouldn't need the cfg here.

Reply:
Ah I didn't expect tidy to complain, ok.

The cfg is required for the opening, but for the closing it's the same content and wouldn't matter for the bootstrap bump though.